tests/e2e/provisioning/pkg/client/broker/broker_client.go

```diff
@@ -132,6 +135,22 @@ func (c *Client) DeprovisionRuntime() (string, error) {
 	return response.Operation, nil
 }
 
+func (c *Client) GlobalAccountID() string {
+	return c.globalAccountID
+}
+
+func (c *Client) InstanceID() string {
+	return c.instanceID
+}
+
+func (c *Client) SubAccountID() string {
+	return c.subAccountID
+}
+
+func (c *Client) GetClusterName() string {
```

Comment:
Why `Get` in the function name? Notice other getter functions does not follow this pattern?

Reply:
removed `Get`